tasks/main.yml

```diff
@@ -29,6 +29,7 @@
     - path: "/etc/swift"
       owner: root
       group: root
+    - path: "/etc/oio/sds/{{ openio_oioswift_namespace }}/watch"
```

Comment:
why not: `{{ openio_oioswift_sysconfig_dir }}/watch` ?

Reply:
I don't know, I copypasted from another project (rawx?). I'll do the change.